src/hooks/dialogmote/document/useReferatDocument.ts

```diff
@@ -152,9 +152,12 @@ export const useReferatDocument = (
   ): DocumentComponentDto[] => {
     const documentComponents: DocumentComponentDto[] = [];
     if (values.standardtekster && values.standardtekster.length > 0) {
+      const standardtekstKeys = Array.isArray(values.standardtekster)
```

Comment:
`standardtekster` på `values` skal være av typen `StandardtekstKey[]`, men virker som react hook form endrer til ren `StandardtekstKey` når lista bare har ett element. Skjønner ikke helt hvorfor det skjer, men dette ser i hvert fall ut til å kunne løse problemet for referat som krasjer i prod.